test/cmdlineTests/asm_json_export_yul_without_debug/output

```diff
@@ -0,0 +1,3 @@
VERT","source":-1},{"begin":765,"end":823,"name":"tag","source":-1,"value":"6"},{"begin":765,"end":823,"name":"JUMPDEST","source":-1},{"begin":906,"end":907,"name":"PUSH","source":-1,"value":"1"},{"begin":859,"end":908,"name":"SLOAD","source":-1},{"begin":855,"end":857,"name":"DUP4","source":-1},{"begin":848,"end":909,"name":"MSTORE","source":-1},{"begin":945,"end":947,"name":"PUSH","source":-1,"value":"20"},{"begin":941,"end":943,"name":"DUP4","source":-1},{"begin":934,"end":948,"name":"RETURN","source":-1},{"begin":991,"end":1349,"name":"tag","source":-1,"value":"4"},{"begin":991,"end":1349,"name":"JUMPDEST","source":-1},{"begin":1036,"end":1047,"name":"CALLVALUE","source":-1},{"begin":1033,"end":1066,"name":"ISZERO","source":-1},{"begin":1033,"end":1066,"name":"PUSH [tag]","source":-1,"value":"7"},{"begin":1033,"end":1066,"name":"JUMPI","source":-1},{"begin":1061,"end":1063,"name":"DUP2","source":-1},{"begin":1057,"end":1059,"name":"DUP3","source":-1},{"begin":1050,"end":1064,"name":"REVERT","source":-1},{"begin":1033,"end":1066,"name":"tag","source":-1,"value":"7"},{"begin":1033,"end":1066,"name":"JUMPDEST","source":-1},{"begin":1127,"end":1129,"name":"DUP2","source":-1},{"begin":1122,"end":1123,"name":"PUSH","source":-1,"value":"3"},{"begin":1118,"end":1124,"name":"NOT","source":-1},{"begin":1102,"end":1116,"name":"CALLDATASIZE","source":-1},{"begin":1098,"end":1125,"name":"ADD","source":-1},{"begin":1094,"end":1130,"name":"SLT","source":-1},{"begin":1091,"end":1149,"name":"ISZERO","source":-1},{"begin":1091,"end":1149,"name":"PUSH [tag]","source":-1,"value":"8"},{"begin":1091,"end":1149,"name":"JUMPI","source":-1},{"begin":1144,"end":1146,"name":"DUP2","source":-1},{"begin":1140,"end":1142,"name":"DUP3","source":-1},{"begin":1133,"end":1147,"name":"REVERT","source":-1},{"begin":1091,"end":1149,"name":"tag","source":-1,"value":"8"},{"begin":1091,"end":1149,"name":"JUMPDEST","source":-1},{"begin":1190,"end":1192,"name":"DUP2","source":-1},{"begin":1184,"end":1193,"name":"SLOAD","source":-1},{"begin":1238,"end":1240,"name":"PUSH","source":-1,"value":"40"},{"begin":1232,"end":1241,"name":"MLOAD","source":-1},{"begin":1281,"end":1283,"name":"DUP2","source":-1},{"begin":1273,"end":1279,"name":"DUP2","source":-1},{"begin":1266,"end":1284,"name":"MSTORE","source":-1},{"begin":1324,"end":1326,"name":"PUSH","source":-1,"value":"20"},{"begin":1316,"end":1322,"name":"DUP2","source":-1},{"begin":1309,"end":1327,"name":"RETURN","source":-1},{"begin":611,"end":1349,"name":"tag","source":-1,"value":"2"},{"begin":611,"end":1349,"name":"JUMPDEST","source":-1},{"begin":611,"end":1349,"name":"POP","source":-1},{"begin":557,"end":1367,"name":"POP","source":-1},{"begin":508,"end":1367,"name":"tag","source":-1,"value":"1"},{"begin":508,"end":1367,"name":"JUMPDEST","source":-1},{"begin":1394,"end":1395,"name":"PUSH","source":-1,"value":"0"},{"begin":1391,"end":1392,"name":"DUP1","source":-1},{"begin":1384,"end":1396,"name":"REVERT","source":-1}]}},"sourceList":[]}
```

Comment:
`source` should not be `-1` here. Also `sourceList` should contain the filename, e.g. `output` in this case.